CMakeLists.txt

```diff
@@ -159,16 +119,9 @@ else()
     message(STATUS "Setting logging level to '${LOGGING_LEVEL}'")
 endif(NOT LOGGING_LEVEL)
 
-target_compile_definitions(rmm PUBLIC SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${LOGGING_LEVEL})
-
-target_link_libraries(rmm PUBLIC spdlog::spdlog_header_only)
-
 ###################################################################################################
 # - install targets -------------------------------------------------------------------------------
 
-install(TARGETS rmm
-        DESTINATION lib)
-
```

Comment:
This probably doesn't much matter right now, since installation only works partially, and there won't be an actual lib. I might add this back eventually, even though there still won't be a library, but there'll be a configuration on how to use it in a downstream project.

Reply:
re-added